server/api/actions/actions.api.spec.js

```diff
@@ -119,16 +120,14 @@ describe('Actions API', function() {
     let action, theme;
     beforeEach(async function() {
       action = await actionFixtures.action();
-      await action.load('theme');
-      theme = getExpectedTheme(action.related('theme'));
     });
 
     it('should retrieve an action with all its relations', async function() {
       const res = await api.retrieve(`/actions/${action.get('api_id')}`);
-      expectActions(res.body, getExpectedAction(action, { theme: theme }), {
+      await expectActions(res.body, getExpectedAction(action, { theme: theme }), {
```

Comment:
`{ theme: theme }` does nothing. Also remove the now unused `theme` variable declaration above.

Reply:
Hum... removing `{ theme: theme }` indeed does not fail the test.

But... I don't understand why 😅 The body contains an action that should have a `theme` property. But the `getExpectedAction` does not return an object with a `theme` property, by default. Hence my believing that I had to pass changes to `getExpectedAction`.

Reply:
Because `expectActions` does nothing with the `theme` property in its `expected` argument. You do the theme expectation manually on the next line.